internal/app/tfsec/checks/azu017.go

```diff
@@ -91,6 +91,9 @@ func init() {
 			}
 
 			for _, rule := range securityRules {
+				if rule.HasChild("access") && rule.GetAttribute("access").Contains("Deny") {
```

Comment:
```suggestion
				if rule.HasChild("access") && rule.GetAttribute("access").Equals("Deny") {
```
Nitpick  - Just looking at the docs in Azure it looks like this can only be `Allow` or `Deny` so lets be explicit here. It doesn't look like `deny` is an option - is that correct?